Source/WarpX.cpp

```diff
@@ -219,6 +219,8 @@ bool WarpX::do_device_synchronize = true;
 bool WarpX::do_device_synchronize = false;
 #endif
 
+bool WarpX::roundrobin_sfc = false;
```

Comment:
We need to add control for this from PICMI, too.

Reply:
done